src/acquisition/covidcast/csv_importer.py

```diff
@@ -92,10 +92,10 @@ def find_issue_specific_csv_files(scan_dir, glob=glob):
         issue_date_value = int(issuedir_match.group(2))
         issue_date = CsvImporter.is_sane_day(issue_date_value)
         if issue_date:
-          logger.info('processing csv files from issue date: "' + str(issue_date) + '", directory', path)
+          logger.info(event='processing csv files from issue', detail=issue_date, file=path)
           yield from CsvImporter.find_csv_files(path, issue=(issue_date, epi.Week.fromdate(issue_date)), glob=glob)
         else:
-          logger.warning(event='invalid issue directory day', detail=issue_date_value, file=path)
+          logger.info(event='invalid issue directory day', detail=issue_date_value, file=path)
```

Comment:
why is this changed to info?

Reply:
when we use warning with the structure logging previously like this:
`logger.warning(event='invalid issue directory day', detail=issue_date_value, file=path)`
 the unit test will not pass.

Reply:
My fault, I change it back.